streams/src/main/java/org/apache/kafka/streams/kstream/JoinWindows.java

```diff
@@ -76,22 +76,25 @@
     /** Maximum time difference for tuples that are after the join tuple. */
     public final long afterMs;
 
-    private final Duration grace;
+    private final long graceMs;
```

Comment:
This might be controversial. I change it, to reuse the return type of `ApiUtils.validateMillisecondDuration`. (Similar in other XxxWindows classes).

Thoughts?

Reply:
It is fine to me.